civihr_default_theme/scss/components/_panel.scss

```diff
@@ -250,6 +250,7 @@ form {
   }
 }
 
+.my-details-view-panel,
 .chr_panel--my-details__data-group__content {
```

Comment:
is `.chr_panel--my-details__data-group__content` even used anymore, given all the changes done to the page? If it's not used anymore, wouldn't it make sense to replace it with the new class, instead of leaving it there?

Reply:
As I can't be 100% sure this class it not being used anywhere else I did not delete it, otherwise I could be affecting style in other pages.

Reply:
As this is a custom class, it has to be somewhere in the code. So please search in the project, if it is not used then please remove it.
PS: I searched and did not find it. So it can be removed.